pkg/cmd/internal/session/session_func.go

```diff
@@ -78,7 +78,9 @@ func ToOptions(flags *pflag.FlagSet) (session.Options, error) {
 		}
 	}
 
+	revert := commandName == "develop"
```

Comment:
Just wondering if we can't have an annotation on the cmd itself defining it's revertable?

Reply:
Ah yea, that's not a bad idea.

Reply:
![go-for-it](https://media.giphy.com/media/8vLrJRCVhvIa0a34te/giphy.gif)

Reply:
https://github.com/maistra/istio-workspace/pull/492/commits/620b3b6b52f88b6807d3b6ca84f66822fb32df85